package.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
@Rekard0 told me that we don't want to have a monorepo like package.jon file @mathewmeconry. I wasn't aware. If we also don't want to have this for eslint/prettier (which should happen globally and in a unified manner), I can put the dependencies back to where they came from.